plugin/web/extensions/sidebar/modals/datacatalog/components/hooks.ts

```diff
@@ -197,6 +205,10 @@ export default () => {
         setCatalogProjectName(e.data.payload.catalogProjectName);
         setPublishToGeospatial(e.data.payload.enableGeoPub);
         setTemplates(e.data.payload.templates);
+        if (e.data.payload.filter) {
+          setFilter(e.data.payload.filter);
+          postMsg({ action: "saveFilter", payload: { filter: e.data.payload.filter } });
```

Comment:
This postMsg isn't necessary. `e.data.payload.filter` comes from th filter that the `saveFilter` action saves to.

Reply:
Done!